esp/services/ws_workunits/ws_workunitsQuerySets.cpp

```diff
@@ -3816,11 +3816,11 @@ bool CWsWorkunitsEx::onWUQueryGetSummaryStats(IEspContext& context, IEspWUQueryG
             return true;
         }
 
-        if (getEspLogLevel() >= LogMax)
         {
             StringBuffer sb;
-            toXML(queryAggregates, sb);
-            DBGLOG("getQueryStats(): '%s' => '%s'", control.str(), sb.str());
+            if (!queryLogMsgManager()->rejectsCategory(LegacyMsgCatMax))
+                toXML(queryAggregates, sb);
+            LOG(LegacyMsgCatMax, "getQueryStats(): '%s' => '%s'", control.str(), sb.str());
```

Comment:
I would put the LOG in the same if with the toXML.  No reason to log the empty string.